### PR TITLE
[ WIP ] Add documentation

### LINKED ISSUE #212: Writing documentation for the project
As the project is getting big and will continue to be, we need proper documentation for control flow, getting started, where to look for when you need to find something and the likes.

Proper documentation would surely help everybody to get along and make the project more welcoming.

How/where to write the docs can be discussed :thinking:. We can make a `documentation.md` which will contain all the information with proper links (wherever needed) or we can use something like `jsdocs` to create an HTML page for documentation (it will automatically generate documentation as an HTML page from comments provided in the code)...

Open for discussion!

### PR DESCRIPTION
**Pull request by Rishabh570**
Fixes #212 
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by Rishabh570** [comment]
Hi @jywarren, this is how documentation part looks,

docs

video guide link is not working as of now because there are some server issues at the [terminalizer's](https://terminalizer.com) part. I'll for them to get resolved or I'll have to find another way of hosting the video (maybe youtube) if that doesn't get resolved soon.
There is room for improvements here, please take a look :)

**Comment by jywarren** [comment]
Also, what if we added a badge specifically for having a welcome page? 

Like:



`https://img.shields.io/badge/newcomers-welcome-pink.svg`

File: README.md [hunk 1]
```diff
@@ -1,82 +1,69 @@
...
+[![Code of Conduct](https://img.shields.io/badge/code-of%20conduct-green.svg)](https://publiclab.org/conduct)
+[![first-timers-only-friendly](http://img.shields.io/badge/first--timers--only-friendly-lightgray.svg?style=flat-square)](https://code.publiclab.org#r=all)
+[![Join the chat at https://publiclab.org/chat](https://img.shields.io/badge/chat-in%20different%20ways-blue.svg)](https://publiclab.org/chat)
+[![visit here](https://img.shields.io/badge/live_at-URL-yellow)](https://code.publiclab.org/)
+[![GitHub license](https://img.shields.io/github/license/publiclab/community-toolbox)](https://github.com/publiclab/community-toolbox/blob/main/LICENSE)
+[![GitHub stars](https://img.shields.io/github/stars/publiclab/community-toolbox)](https://github.com/publiclab/community-toolbox/stargazers)
 
 
-## Embed an issues list
+## Project Overview
```
[the remark this answers is not in the document]
Comment (Rishabh570) [reply README.md]: Yes, that would super helpful...I'll do it :)

**Comment by Rishabh570** [comment]
> Also, what if we added a badge specifically for having a welcome page?

Do you mean this badge would be present in the readme if the org has a welcome page?

**Comment by jywarren** [comment]
Yeah, it'd be cool, no?

**Comment by Rishabh570** [comment]
Yeah, I'll add it in the readme file :)

**Comment by Rishabh570** [comment]
Hi @jywarren, I've updated the link, it now points to [youtube](https://youtu.be/wkC2AgLiaTM). Also, the table of contents and badge are added...please take a look.

**Comment by jywarren** [comment]
This looks awesome. Let's merge!

**Pull request merged**